### DISCUSSION TITLE
Hot tub with natural hot water.

### DISCUSSION THREAD

**Discussion by Volvoinn** [tagyoureit/nodejs-poolController#643] [General]
Hi there. I have been reading about nodejs-poolcontroller.
I like many things there and I would like to adjust it to my needs.
I live in Iceland and here we are able to use natural hot water for our tub and I do. No heater, no pump, just a mixing valve.
I know what I want to do but I'm not sure which way to go. So my question is this, since you have the know how to make this controller, would it be possible to get your advice on my thoughts and guide me in the right direction.?

**Comment by rstrouse** [comment]
Yes and you can simulate your setup to get an idea of how it works.  Install njsPC, dashPanel, and REM on any platform.
https://github.com/tagyoureit/nodejs-poolController/wiki/Quick-Start-Guide

Once you have done that set it up as a Nixie Single body system and install a gas heater in the configuration pages.  The gas heater configuration should allow you to set up a relay to move the valve.  You can also simulate a water temp sensor.  If you use the solar heater function you can use two temp sensors that will monitor the source temperature and the desired temperature to determine whether it makes sense to activate the valve.

You should probably consider a Raspberry Pi with a relay hat as your final install but this should allow you to try it out.  Your first step is to get the software set up and we can walk you through the final stages.
